### PR TITLE
Adding CI

### PR DESCRIPTION
**Pull request by abhogle**

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -1,16 +1,18 @@
 const express = require('express');
+require('dotenv').config();
```
Comment (TandyN) [inline server/index.js:2]: You able to put this at the very top? Kind of feels out of place, just an opinion.

File: .eslintignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
+client/dist/*.js
+webpack.config.js
```
Comment (TandyN) [inline .eslintignore:2]: Add white space below please.

File: server/index.js (hunk 1 above)
Comment (abhogle) [reply server/index.js:2]: Recommended usage of this package is 'As early as possible in your application, require and configure dotenv.' I suppose to allow for variables to change when deployed.

Comment (abhogle) [reply server/index.js:2]: https://www.npmjs.com/package/dotenv

**Review by TandyN — APPROVED** [review]
You've configured your yml file for GitLab CI & I can see your tests passing. I also see that you're using the dotenv library. Everything checks out.

File: server/index.js (hunk 1 above)
Comment (TandyN) [reply server/index.js:2]: Does the code break when requiring dotenv before initializing express? Since express isn't quite invoking anything yet.

**Comment by abhogle** [comment]
dotenv is essentially loading env variables on the respective file. Loading
it before express does not conflict and runs to code when executed
appropriately.

AB

**Pull request merged**